awx/main/dispatch/worker/callback.py

```diff
@@ -111,7 +110,6 @@ def _save_event_data():
                 except (OperationalError, InterfaceError, InternalError):
                     if retries >= self.MAX_RETRIES:
                         logger.exception('Worker could not re-establish database connectivity, shutting down gracefully: Job {}'.format(job_identifier))
-                        os.kill(os.getppid(), signal.SIGINT)
```

Comment:
Discussed w/ @chrismeyersfsu, we don't need this anymore, and it's causing some nuanced problems.